### PR TITLE
feat(icons-list): Adding a page displaying all the available icons - OEL-1684

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

**Comment by planctus** [comment]
Cool!! But can we align those icons..? It looks a bit messy right now :)

File: src/compositions/bcl-colors/colors.story.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ const applyArgs = (data, args) => {
 };
 
 export default {
-  title: "Colors",
+  title: "Resources/Colors",
   decorators: [withCode],
```
Comment (planctus) [inline src/compositions/bcl-colors/colors.story.js:130]: i don't think we need this panel here.. ;)

File: src/compositions/bcl-icons-list/icons-list.story.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { correctPaths } from "@openeuropa/bcl-story-utils";
+import icons_list from "@openeuropa/bcl-icons-list/icons-list.html.twig";
+import icons from "@openeuropa/bcl-theme-default/src/icons/icons.js";
+import defaultSprite from "@openeuropa/bcl-theme-default/icons/bcl-default-icons.svg";
+
+let list = icons.map((icon) => icon.substring(0, icon.length - 4));
+
+console.log(list);
```
Comment (planctus) [inline src/compositions/bcl-icons-list/icons-list.story.js:8]: i stil have to understand the reason why the linter is not complaining when we forget a console.log().. :)

File: src/compositions/bcl-icons-list/icons-list.story.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+export default {
+  title: "Resources/Icons",
+  parameters: {
+    layout: "fullscreen",
+    controls: { disable: true },
+  },
+};
+
+export const Default = () =>
+  icons_list(correctPaths({ icons: list, path: defaultSprite }));
```
Comment (planctus) [inline src/compositions/bcl-icons-list/icons-list.story.js:19]: this confuses me a bit, why do we need to use correctPaths() if we are importing the sprite, or viceversa, why do we import the sprite if we set the right path through the utility function..?

**Review by planctus — CHANGES_REQUESTED** [review]
A few things and we are ok

**Comment by tibi2303** [comment]
@planctus done, thx

**Review by planctus — APPROVED** [review]

**Pull request merged**